lib/cisco_node_utils/bgp_af.rb

```diff
@@ -228,7 +228,6 @@ def advertise_l2vpn_evpn
     end
 
     def advertise_l2vpn_evpn=(state)
-      Feature.nv_overlay_enable
```

Comment:
@chrisvanheuveln Can you comment on why/if `Feature.nv_overlay_enable` is still needed here?